tests/plugins/test_koji_import.py

```diff
@@ -1345,3 +1350,46 @@ def test_koji_import_set_media_types(self, tmpdir, os_env, version):
             assert image['media_types'] == version
         else:
             assert 'media_types' not in image.keys()
+
+    @pytest.mark.parametrize('digests', [
+        [],
+        [ManifestDigest(v2_list='sha256:e6593f3e')],
+    ])
+    def test_koji_import_set_manifest_list_info(self, tmpdir, os_env, digests):
+        session = MockedClientSession('')
+        tasker, workflow = mock_environment(tmpdir,
+                                            name='ns/name',
+                                            version='1.0',
+                                            release='1',
+                                            session=session,
+                                            docker_registry=True)
+        workflow.postbuild_results[PLUGIN_GROUP_MANIFESTS_KEY] = digests
+        orchestrate_plugin = workflow.plugin_workspace[OrchestrateBuildPlugin.key]
+        orchestrate_plugin[WORKSPACE_KEY_BUILD_INFO]['x86_64'] = BuildInfo()
+        runner = create_runner(tasker, workflow)
+        runner.run()
+
+        data = session.metadata
+        assert 'build' in data
+        build = data['build']
+        assert isinstance(build, dict)
+        assert 'extra' in build
+        extra = build['extra']
+        assert isinstance(extra, dict)
+        assert 'image' in extra
+        image = extra['image']
+        assert isinstance(image, dict)
+        if digests:
+            assert 'index' in image.keys()
+            expected_results = {}
+            expected_results['tags'] = [tag.tag for tag in workflow.tag_conf.images]
+            pullspec = "docker.example.com/hello-world@{0}".format(digests[0].v2_list)
```

Comment:
This is incorrect! It should be "docker.example.com/myproject/hello-world@{0}". Without the namespace part included, the image is not pullable.

Reply:
Fixed.